presto-hive-common/src/main/java/org/apache/hadoop/fs/HadoopExtendedFileSystem.java

```diff
@@ -65,6 +65,12 @@ public FileSystem getRawFileSystem()
         return fs;
     }
 
+    @Override
+    public String getScheme()
```

Comment:
The method in its parent class `FileSystem` just throws an UnsupportedOperationException. We have to override it here to make HoodieWrapperFileSystem happy since it will call this method.